utils/python/CIME/test_status.py

```diff
@@ -31,10 +31,11 @@
 
 # The statuses that a phase can be in
 TEST_PEND_STATUS = "PEND"
+TEST_RUN_STATUS = "RUN"
 TEST_PASS_STATUS = "PASS"
 TEST_FAIL_STATUS = "FAIL"
 
-ALL_PHASE_STATUSES = [TEST_PEND_STATUS, TEST_PASS_STATUS, TEST_FAIL_STATUS]
+ALL_PHASE_STATUSES = [TEST_PEND_STATUS, TEST_RUN_STATUS, TEST_PASS_STATUS, TEST_FAIL_STATUS]
```

Comment:
There's a lot of code that's highly dependent on the states in TestStatus. I'm pretty sure most of the code is not prepared to handle this additional state.